module/az.c

```diff
@@ -53,48 +49,39 @@
 #define RESPONSE_HEADER_LENGTH 1024 // Response Header Length // Azure sends on average 592 bytes
 
 // PUT REQUEST HEADER
-//PATH/HOST/Lease/ContentLength/Range-Start/Range-End/Date/AccountName/AuthToken
-static const char *put_request_head = "PUT %s?comp=page HTTP/1.1\r\n"
+//PATH/Sas/HOST/Sas/Lease/ContentLength/Range-Start/Range-End/Date/AccountName/AuthToken
```

Comment:
A general question, it looks dysk now only work with sas key. If the key expires, how can it be renewed?

Reply:
sas lifetime has nothing to do with storage key. You can rotate storage key and sas is still valid.

Reply:
I would keep this outside the kernel module, handled by "outside" possible an orchestrator?